deepchecks/core/serialization/check_result/json.py

```diff
@@ -129,9 +127,9 @@ def handle_dataframe(
     def handle_callable(cls, item: t.Callable, index: int, **kwargs) -> t.Dict[str, t.Any]:
         """Handle callable."""
         return {
-            'type': 'images',
+            'type': 'html',
             'payload': [
-                base64.b64encode(buffer.read()).decode('ascii')
+                imagetag(base64.b64encode(buffer.read()))
```

Comment:
why? what if the user wants to process those images in a different environment, with help of other tools and not only to display thems in the browser? he would need to parse that image tag then/